webpack.common.js

```diff
@@ -49,17 +49,17 @@ module.exports = {
         test: /\.tsx?$/,
         use: "ts-loader",
       },
-      // TODO: string-replace-loader has issues with new webpack version
-      // {
-      //     test: /ejs\.js$/,
-      //     loader: 'string-replace-loader',
-      //     options: {
-      //         search: "(new Function('return this;'))().Promise",
-      //         replace: "(function() { return this ? this : typeof self !== 'undefined' ? self : undefined})() || Function('return this')()",
-      //         flags: 'g',
-      //         strict: true
-      //     }
-      // },
+      {
+        test: /backgroundPage\.js$/,
+        loader: "string-replace-loader",
```

Comment:
I tested this plugin before but it is not working for some reason. I still getting this 'unsafe-eval' error. 
![Screenshot_20230221_125246](https://user-images.githubusercontent.com/36774944/220338026-841e6690-70d3-4ff0-a7b7-f63c2fc1ad98.png)

Reply:
Yes, seems it doesn't work.
I have cached version.